spec/machine/ARM/MachineOps.thy

```diff
@@ -454,14 +454,12 @@ where
 
 section "Memory Clearance"
 
-text \<open>Clear memory contents to recycle it as user memory\<close>
+text \<open>Clear memory contents to recycle it as user memory. Do not yet flush the cache.\<close>
```

Comment:
does this comment exist on ARM_HYP? didn't change there, only ARM+AARCH64

Reply:
Well spotted, have updated.